src/main/java/frc/robot/Controls.java

```diff
@@ -15,9 +20,61 @@ public class Controls {
   @SuppressWarnings("UnusedVariable")
   private final Subsystems s;
 
+  // Swerve stuff
+  private double MaxSpeed =
+      BonkTunerConstants.kSpeedAt12Volts.in(MetersPerSecond); // kSpeedAt12Volts desired top speed
+  private double MaxAngularRate =
+      RotationsPerSecond.of(0.75)
+          .in(RadiansPerSecond); // 3/4 of a rotation per second max angular velocity
+
+  /* Setting up bindings for necessary control of the swerve drive platform */
+  private final SwerveRequest.FieldCentric drive =
+      new SwerveRequest.FieldCentric()
+          .withDeadband(MaxSpeed * 0.1)
+          .withRotationalDeadband(MaxAngularRate * 0.1) // Add a 10% deadband
+          .withDriveRequestType(
+              DriveRequestType.OpenLoopVoltage); // Use open-loop control for drive motors
+  private final SwerveRequest.SwerveDriveBrake brake = new SwerveRequest.SwerveDriveBrake();
+  private final SwerveRequest.PointWheelsAt point = new SwerveRequest.PointWheelsAt();
+
+  private final Telemetry logger = new Telemetry(MaxSpeed);
+
   public Controls(Subsystems s) {
     driverController = new CommandXboxController(DRIVER_CONTROLLER_PORT);
     operatorController = new CommandXboxController(OPERATOR_CONTROLLER_PORT);
     this.s = s;
+    configureDrivebaseBindings();
+  }
+
+  private void configureDrivebaseBindings() {
```

Comment:
```suggestion
  private void configureDrivebaseBindings() {
    if (s.drivebaseSubsystem == null) {
      // Stop running this method
      return;
    }
```
We could also do the null check in the constructor or check the enabled constant, but this is the most direct way to not configure drivebase bindings when we don't have a drivebase subsystem object.

Reply:
Added. Commit will be pushed momentarily